src/connect.rs

```diff
@@ -444,14 +461,134 @@ impl Service<Uri> for Connector {
     }
 }
 
+/// Extra transport layer information applicable to HTTPS.
+#[derive(Clone)]
+pub struct HttpsInfo {
```

Comment:
Some thoughts:

- Should this perhaps live in `reqwest::tls`?
- I almost wanted to suggest the name `ServerCertificates`, but that doesn't seem quite right (other info could be present too, and server certificates can be set in the `ClientBuilder`). This name does follow the pattern of `HttpInfo`... it just felt a little generic. I dunno, I don't have a great suggestion for my feeling.

Reply:
That's fair, and I do actually think `HttpInfo` is a bit of a misnomer.  How about `TlsInfo`, living in `reqwest::tls`?  Less generic, broad enough to also cover other TLS-related data one might need in the future.

Reply:
Does it include client TLS info too? Or just from the server?

Reply:
Things like TLS version or cipher suite that folks might conceivably want to add in the future are properties of the connection.  Not sure if splitting info up into multiple extensions is worth it.

Reply:
Sounds good to me :)